packages/story-editor/src/utils/storyUpdates.ts

```diff
@@ -22,20 +22,35 @@
  */
 import cleanForSlug from './cleanForSlug';
 
+interface updateStoryProps {
+  properties: {
+    slug: string;
+  };
+}
+
+interface updateSlugProps {
+  currentSlug: string;
+  currentTitle: string;
+  updateStory: (props: updateStoryProps) => void;
```

Comment:
See the `updateStory` in the `StoryProvider`, these types should probably match since the `updateStory` comes from the story provider.